src/main/java/uk/gov/justice/hmpps/prison/service/transfer/PrisonTransferService.kt

```diff
@@ -117,7 +117,7 @@ class PrisonTransferService(
             OffenderProgramEndReason.TRF.code
           )
           trustAccountService.createTrustAccount(
-            booking = this, movementOut = toCourtMovement, movementIn = createdMovement
+            booking = this, fromAgency = toCourtMovement.fromAgency, movementIn = createdMovement
```

Comment:
switched to just passing in the fromAgency rather than whole movement since that is all service needs - see not below that that might not be necessary either ....